commands/configcmd.go

```diff
@@ -129,6 +126,10 @@ func SetCommand(parent string) *cobra.Command {
 	fieldmeta.SetShortHandRef(ShortHandRef)
 	kustomizeCmd := configcobra.Set(parent)
 	setCmd := *kustomizeCmd
+	kustomizeCmd.Short = cfgdocs.SetShort
```

Comment:
Why do we need special handling here for the set command?

Reply:
From my understanding, this is due to gcloud auto-setters for which the code can exit in kpt only, the command is handled in a different way. We should consider making it consistent with other commands. This one off would lead to issues in future.